src/components/content/common/ocl/DisplayOclData.tsx

```diff
@@ -118,6 +117,33 @@ function DisplayOclData({ ocl }: { ocl: Ocl }): React.JSX.Element | string {
                                 {ocl.eula ? <AgreementText eula={ocl.eula} /> : <span>Not Provided</span>}
                             </Descriptions.Item>
                         </Descriptions>
+                        <h3 className={catalogStyles.catalogDetailsH3}>
+                            <ProfileOutlined />
+                            &nbsp;Deployment Information
+                        </h3>
+                        <Descriptions column={2} bordered className={oclDisplayStyles.oclDataInfoTable}>
+                            <Descriptions.Item label='Kind'>
+                                {
+                                    <img
+                                        src={
+                                            ocl.deployment.deployerTool.kind.valueOf() === 'terraform'
```

Comment:
better to move this to a separate small function instead of tenary operator. In future there can be new deployers. So bettter to have something similar to switch case.

Reply:
fixed